docs/snippets/common/main-config-addons.ts.mdx

```diff
@@ -0,0 +1,22 @@
+```ts
+// .storybook/main.ts
+
+// Replace your-framework with the framework you are using (e.g., react-webpack5, vue3-webpack5)
+import type { StorybookConfig } from '@storybook/your-framework';
+
+const config: StorybookConfig = {
+  framework: '@storybook/vue3-vite',
```

Comment:
I wonder if we should use `'@storybook/your-framework'` here too, to match the comment above?

Reply:
@kylegach we should, If you could update the examples in your pull request and I'll pick on them and update any other references that are still around. Sounds good?